astroquery/atomic/core.py

```diff
@@ -248,6 +248,10 @@ def query_object_async(self, *, wavelength_range=None, wavelength_type='', wavel
         return response
 
     def _parse_result(self, response):
+
+        if 'ERROR: request form contains no information' in response.text:
```

Comment:
Could we make it more generic?
Maybe either check only for `'ERROR'`, or split the data assignment below, and check for `'ERROR'` in response text if `BeautifulSoup(response.text, features='html5lib').find('pre')` is None before processing it further?

Reply:
Oh, you did both. Thanks!

Reply:
I added after your suggestion.